### PR TITLE
Fix/vpaid integration

### PR DESCRIPTION
**Pull request by carpasse**

### CODE REVIEW

File: packages/video-ad-sdk/src/adContainer/__tests__/VideoAdContainer.spec.js [hunk 1]
```diff
@@ -53,6 +56,7 @@ describe('VideoAdContainer', () => {
     expect(videoAdContainer.videoElement.parentNode).toBe(adContainerElement);
   });
 
+  // eslint-disable-next-line jest/no-disabled-tests
```
Comment (xram) [inline packages/video-ad-sdk/src/adContainer/__tests__/VideoAdContainer.spec.js]: Can this be removed now? It doesn't look like the test is disabled.

File: packages/video-ad-sdk/src/adContainer/helpers/__tests__/createIframe.spec.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
 import createIframe from '../createIframe';
+import supportsSrcdoc from '../supportsSrcdoc';
 
+jest.mock('../supportsSrcdoc');
+// eslint-disable-next-line jest/no-disabled-tests
```
Comment (xram) [inline packages/video-ad-sdk/src/adContainer/helpers/__tests__/createIframe.spec.js]: Can this be removed now? It doesn't look like the test is disabled.

File: packages/video-ad-sdk/src/adContainer/helpers/supportsSrcdoc.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+const supportsSrcdoc = () => 'srcdoc' in document.createElement('iframe');
```
Comment (Fetz) [inline packages/video-ad-sdk/src/adContainer/helpers/supportsSrcdoc.js]: is this being executed multiple times? if yes then you can check once

File: packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js [hunk 4]
```diff
@@ -57,6 +64,119 @@ describe('runWaterfall', () => {
     jest.resetAllMocks();
   });
 
+  describe('after fetching Vast response', () => {
+    test('must call onError is Vast response is undefined', async () => {
```
Comment (xram) [inline packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js]: ~`is`~ `if`

File: packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js [hunk 5]
```diff
@@ -57,6 +64,119 @@ describe('runWaterfall', () => {
...
+      await runWaterfall(adTag, placeholder, {
+        ...options,
+        onError
+      });
+
+      expect(onError).toHaveBeenCalledTimes(1);
+      expect(onError.mock.calls[0][0].message).toBe('Invalid VastChain');
+    });
+
+    test('must call onError is Vast response is an empty array', async () => {
```
Comment (xram) [inline packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js]: ~`is`~ `if`

File: packages/video-ad-sdk/src/adContainer/__tests__/VideoAdContainer.spec.js (hunk 1 above)
Comment (carpasse) [reply packages/video-ad-sdk/src/adContainer/__tests__/VideoAdContainer.spec.js]: 👍

File: packages/video-ad-sdk/src/adContainer/helpers/__tests__/createIframe.spec.js (hunk 2 above)
Comment (carpasse) [reply packages/video-ad-sdk/src/adContainer/helpers/__tests__/createIframe.spec.js]: 👍

File: packages/video-ad-sdk/src/adContainer/helpers/supportsSrcdoc.js (hunk 3 above)
Comment (carpasse) [reply packages/video-ad-sdk/src/adContainer/helpers/supportsSrcdoc.js]: 👍

File: packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js (hunk 5 above)
Comment (carpasse) [reply packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js]: 👍

File: packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js (hunk 4 above)
Comment (carpasse) [reply packages/video-ad-sdk/src/runner/__tests__/runWaterfall.spec.js]: 👍

**Pull request merged**